resources/views/setup/app-settings.blade.php

```diff
@@ -14,7 +14,7 @@
         <div class="field">
           <label class="label">Website Name</label>
           <div class="control">
-            <input class="input" type="text" placeholder="Meetable" value="{{ session('setup.app_name', 'Meetable') }}" name="app_name">
+            <input class="input" type="text" placeholder="Meetable" value="{{ session('setup.app_name', env('APP_NAME', 'Meetable')) }}" name="app_name">
```

Comment:
I'll back out of this change

Reply:
There probably won't be a case where the `APP_NAME` is already set, but I guess this doesn't hurt.

Reply:
You explicitly set it in index.php. I'll back it out 😉